### DISCUSSION TITLE
ERP values in srsRAN gNB

### DISCUSSION THREAD

**Discussion by MamdouhEgy** [srsran/srsRAN_Project#678] [General Help]
Hello Everyone,

I successfully created a 5G lab based on srsRAN + Open5gs and bladeRF 2.0 A4. I just want to confirm where exactly the ERP values are located.

The formula for ERP (dBm) is:

ERP (dBm) = Transmitter Power (dBm) + Antenna Gain (dBi) - Losses (dB)

I already know that some configurations (tx_gain and rx_gain) are in the `gnb_rf_b200_tdd_n78_20mhz.yml` file, but I want to be certain that I am using the correct allowed ERP value of 200 mW in my case.

Thank you.

**Comment by pgawlowicz** [comment]
The best would be to measure the tx power with a spectrum analyzer.
